package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "csbot",
-	"version": "0.8.0",
+	"version": "0.7.1",
```

Comment:
Bumping down the version?

Reply:
Following changelog

Reply:
I think 8.1 is what it should be right now. I forgot to update a major version in my PR iirc

Reply:
0.8.2 if you count Plyb's PR that was merged in just a bit ago

Reply:
We can just group all those changes together into one update.

Reply:
How's that?

Reply:
I swear, I'm working on a proper automation for this 😅